web/src/features/hfiCalculator/slices/hfiPrepSlice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+const MAX_PREP_DAYS = 5
```

Comment:
This constant seems to show up in a few places. Also in PrepDaysDropdown and there's a NUM_WEEK_DAYS in hfiCalculator/contants.ts - are they the same constant?

Reply:
Ultimately they are, I'll address this in the redux refactor.